python/tests/backends/test_Quantinuum_LocalEmulation_kernel.py

```diff
@@ -325,28 +325,6 @@ def kernel(thetas: list[float]):
     assert_close(expectation, 0.707)
 
 
-def test_capture_array():
```

Comment:
@schweitzpgi Just confirming if this complex array-capture feature is supported for hardware emulation?